src/lamplib/tasks/run_tests.py

```diff
@@ -110,7 +110,11 @@ def resmoke_test(env, suites, mongo_dir, is_cnats):
         # Default download location for MongoDB binaries.
         env["PATH"] += ":" + os.path.join(mongo_dir, "bin") + ":" + mongo_dir
 
-    evg_venv_dir = os.path.join(os.environ['SCRIPTS_DIR'], "venv")
+    evg_venv_dir = os.path.join(os.environ['LAMP_VENV_DIR'], "venv")
```

Comment:
Oh i'm sorry, I didn't understand how os.environ worked. This will raise a KeyError exception if LAMP_VENV_DIR isn't set. Let's try/catch this., and raise a prettier error message